sign-windows/action.yaml

```diff
@@ -0,0 +1,50 @@
+name: sign-windows
+description:
+  Sign windows files using AzureSignTool
+
+inputs:
+  files-to-sign:
+    description: "List of files to sign"
+    required: true
+  vault-uri:
+    description: "Azure key vault URI"
+    required: true
+  cert-name:
+    description: "Name of certificate on Azure"
+    required: true
+  client-id:
+    description: "Azure signer app clientId"
+    required: true
+  client-secret:
+    description: "Azure signer app clientSecret"
+    required: true
+  tenant-id:
+    description: "Azure signer app tenantId"
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - name: Install viewerutil
+      shell: bash
+      run: which sign-windows || pip install -e "$(dirname ${{ github.action_path }})/.."
+
+    - name: Install AzureSignTool
+      shell: bash
+      run: which azuresigntool || dotnet tool install --global AzureSignTool
+
+    - name: Sign files
+      shell: bash
+      env:
+        FILES_TO_SIGN: ${{ inputs.files-to-sign }}
```

Comment:
It looks as though this and the `for` loop on line 41 won't correctly handle pathnames containing spaces.

Reply:
Ah, you're right. I've moved the multiple-file handling to python to address this and added a test. Thanks!